opa/opa_test.go

```diff
@@ -0,0 +1,34 @@
+package opa
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+func TestFileSecurity(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Opa")
+}
+
+var _ = Describe("Opa", func() {
+	var log *logrus.Entry
+
+	BeforeEach(func() {
+		log = logrus.NewEntry(logrus.New())
+		log.Logger.SetOutput(GinkgoWriter)
+	})
+
+	Describe("Evaluate", func() {
+		It("Should support basic evaluations", func() {
+			inputs := map[string]interface{}{"hello": "world"}
+
+			pass, err := Evaluate(context.Background(), "io.choria.ginkgo", "data.io.choria.ginkgo.allow", inputs, Logger(log), File("testdata/test1.rego"), Trace())
```

Comment:
we could probably assume that for module `io.choria.ginko` that the query `allow` would mean `data.io.choria.ginko.allow` and so the query argument here could be just `allow`? But probably early optimisation we will use this like 4 times